docs/src/content/docs/getting-started/usage.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Usage
+description: Basic usage of `@sheepdog/svelte`.
+---
+
+import { Tabs, TabItem, Code, Aside, LinkCard } from '@astrojs/starlight/components';
+
+The simplest way to use `@sheepdog/svelte` is the following
+
+<Tabs syncKey="examples">
+	<TabItem label="Dot notation">
+	```svelte
+	<script>
+		import { task, timeout } from '@sheepdog/svelte';
+
+    	const myTask = task(async () => {
+    		// async stuff
+    	});
```

Comment:
I think you made a good point earlier about this being confusing. Is it better to use a different task type here that actually displays the dot notation?

Reply:
I'm very thorned about this because on one hand i want to show the simplest possible code...noone will ever use `task.default`...maybe we should just not show the option notation here and just show it below?

Reply:
Maybe that's best. The tabs could be `default` and `option notation`?